README.md

```diff
@@ -1 +1,7 @@
-# updatecli-automation
\ No newline at end of file
+# Updatecli automation
+
+*Note:* This is still work in progress and subject to change.
+
+This project is managed by Rancher Security team and uses [Updatecli](https://github.com/updatecli/updatecli) for automation of security related version bumps.
+
+
```

Comment:
Can you add more basic info. For example to answer questions: 
* Whats extent of action taken by this repo. Does this just open PRs and you have to go merge them like dependabot ? What bot user would do that ?
* I'm confused how to think about this project, whats the "master list" or "directory" here. Is each folder a repo (like promeheus-federator), and then you change things in a repo ? Or is it the opposite where there is a shell item and you work into each repo from there. Seems like former. 
* Do we have a system or guidelines for adding new images. If you are going to do one place, should you try to do all places ? Or do we allow disconnects where shell gets updated different ways in different repos. 
* How do you develop a new item locally or in the repo, is there a dry run ?

Reply:
Those are good and valid points. I was planning to better address the README and to add instructions on a separate PR. Do you prefer for me to address this now or is it fine to wait?

Reply:
There are some things dependabot doesn't do. For example, security handling on non-default branches. I'm curious what we will have dependabot do for us and what we will use updatecli for.

Reply:
Yes, Dependabot has a few limitations. Updatecli will be used in combination with RenovateBot (that @superseb is working on). RenovateBot will handle the update of language dependencies and Updatecli the orchestration of updates (e.g., if image X has a new version, then update it in service Y, then update version of Y in repo Z). Both tools have an overlap in features like updating environment variables, which isn't supported in Dependabot. How we will define this overlap responsibility between Updatecli and RenovateBot is still to be defined, but shouldn't be a concern now, as we need to get the ball rolling sooner than later IMO.

Reply:
@cbron I added a readme. Hopefully I was able to address yours questions.